dev/refresh_source.py

```diff
@@ -434,7 +434,8 @@ def write_deck_run_script(self, repository):
 cd "$d"
 LOG_DIR=${{LOG_DIR:-../logs}}
 
-bash -c "(./start.sh >> '$LOG_DIR/{name}.log') 2>&1\
+npm install >& "$LOG_DIR/deck.log"
```

Comment:
Right, but `refresh_source.py` formats those `{name}` pieces. The one `deck.log` is inconsistent, so I'll change that.

I cloned this branch, then ran 
```
spinnaker/dev/refresh_source.sh --pull_origin --github_user spinnaker
spinnaker/dev/run_dev.sh
```
and it produced the right log files.